tests/test-cases.js

```diff
@@ -872,6 +872,66 @@ var PARSE_TESTS = [
 			errors: []
 		}
 	},
+	{
+		description: "Custom transform function converts all true and false strings to booleans regardless of case",
```

Comment:
I will prefer to have a generic test description that talks about that a transform function can be applied and does not talk about the type of transformations applied